content/posts/open_channel_pre_taproot.md

```diff
@@ -0,0 +1,280 @@
+`bob_node_id`. Their main two goals are: 
+
+1. They want to open a channel between themselves in a trustless way.
+2. They want to be able to advertise their new channel so that the rest of the
+   network can use it for routing.
+
+## Opening the channel
+
+A channel is opened once both parties have the ability to fully sign their
+respective commitment transactions and once the funding transaction is on chain.
+Throughout this explanation, I will use diagrams and colours to illustrate the
+process. In the diagrams, a white background means that the field of the
+transaction is not yet known. A coloured-in field indicates that the field value
+is known. Any green fields are used to highlight any of Alice’s public keys or
+signatures and any blue fields are used to show Bob’s. 
+
+There are three transactions in play for the opening of a transaction. The first
+is the funding transaction which will need to go on chain. The other two are the
+first commitment transactions held by Alice and Bob describing the initial state
+of the channel.
+
+Here are the diagrams used to describe the three transactions. Note that at the
+moment we still don’t know anything about the parameters of the channel and so
+all fields still have a white background. Remember that our end goal is to have
+all these fields coloured in. 
+
+ ![](/openChanPt1/fntx_1.png#center)
+ ![](/openChanPt1/1a_1.png#center)
+ ![](/openChanPt1/1b_1.png#center)
+
+### The `open_channel` message
+
+The first step of the process is Alice deciding that she wants to open a channel
+with Bob. In doing so, she also becomes the funder of the channel. Let’s say
+that Alice decides that she wants to open a 1 BTC channel and wants to
+immediately give Bob half of the channel’s capacity (0.5 BTC). Alice will now
+put together an `open_channel` message that she will send to Bob. Let’s take a
+look at the relevant `open_channel` parameters (I am leaving out any parameters
+that don’t have anything to do with the channel-open phase). You can skip this
+table if you want. It is just a reference for incase you want to check what the
+meaning of a field is.
+
+ ![](/openChanPt1/open_channel_params.png#center)
+
+Let’s now fill in Alice’s values for these fields:
+
+ ![](/openChanPt1/open_chan.png#center)
+
+Alice sends this message over to Bob. 
+
+Since Alice has decided on the type of channel she wants to open (default
+channel type) as well as the channel capacity, she can already piece together
+quite a large part of the funding transaction:
+
+ ![](/openChanPt1/fntx_2.png#center)
+
+Since she knows the capacity of the channel she wants to open, she can choose
+some of her UTXOs to be inputs to the funding transaction. Since Alice does not
+yet know which pub key Bob would want to use for the channel, she cannot yet
+finalise the channel funding output and hence can also not yet produce
+signatures for the inputs.
+
+Since Alice has decided on the channel type (and hence the commitment
```

Comment:
Slightly confusing since the reader (me) doesn't know what types of channels there are. Suspect it's left out because it's irrelevant, but it's the first time in the article "channel type" is mentioned.

Reply:
gonna extend the previous mention of it to say "default as opposed to anchors or taproot"